### DISCUSSION TITLE
I don't understand how it work (PhpDocInfo)

### DISCUSSION THREAD

**Discussion by koraga** [rectorphp/rector#7718] [Q&A]
Hi everybody! 
I do custom rule for my project. I need to add @see tags in my routes file. Example:
<img width="603" alt="Screenshot 2023-01-16 at 2 19 48 PM" src="https://user-images.githubusercontent.com/34651075/212666337-5c67d009-7a85-4e3c-a680-8ceb5f585a10.png">

I did this rule: 
<img width="784" alt="Screenshot 2023-01-16 at 2 15 38 PM" src="https://user-images.githubusercontent.com/34651075/212665588-f768356b-fc12-4b9c-8cc5-eccfa13951c1.png">

But it not work. 
<img width="660" alt="Screenshot 2023-01-16 at 2 17 32 PM" src="https://user-images.githubusercontent.com/34651075/212665922-14a1284b-a2a4-4ac4-acca-3ac930134666.png">

What am I doing wrong?

**Comment by samsonasik** [comment]
You can check the implementation for add `@see` on `AddSeeTestAnnotationRector`

https://github.com/rectorphp/rector-phpunit/blob/main/src/Rector/Class_/AddSeeTestAnnotationRector.php

**Reply by koraga** [reply]
I saw it. My code based on this implementation. 
I use `addPhpDocTagNode($phpDocSeeTagNode);` but my rule still not working

**Reply by samsonasik** [reply]
You may need to get PhpDocInfo from `\PhpParser\Node\Stmt\Expression` statement by define `\PhpParser\Node\Stmt\Expression::class` as node type.
